pytorch/inference/docker/1.10/py3/Dockerfile.graviton.e3.cpu

```diff
@@ -8,7 +8,7 @@ ARG PYTHON_VERSION=3.8.12
 ARG PT_INFERENCE_URL=https://pytorch-ei-binaries.s3.us-west-2.amazonaws.com/r1.10.0_inference/20211117-055259/7af6f8360b1e31f4bad44bdec916cf15efc9b8a5/graviton/torch-1.10.0%2Bcpu-cp38-cp38-manylinux2014_aarch64.whl
 ARG ARMCL_BRANCH=v21.05
 ARG OPEN_MPI_VERSION=4.0.1
-ARG TS_VERSION=0.3.1
+ARG TS_VERSION=0.5.2
```

Comment:
Would you mind updating graviton in a separate PR? will be easier to test this way

Reply:
Created PR here: https://github.com/aws/deep-learning-containers/pull/1608